AY2122S1-CS2103T-T17-3/tp
# Add multiple address book

### Description

Added minimal functionality for handling of multiple address book.
Added commands to add, switch and list the address book.

Command | Action
--- | ---
`ab create {ADDRESSBOOK_NAME}` | create a new address book with name `{ADDRESSBOOK_NAME}`
`ab switch{ADDRESSBOOK_NAME}` | switch to address book with name `{ADDRESSBOOK_NAME}`
`ab list` | list all the address book

### Type of change


### How to test

Starting with only the default address book (../data/addressbook.json)

command | expected behaviour
--- | ---
`ab list` | list out only addressbook
`ab create addressbook` | give error addressbook already exists
`ab create anotherbook` | create and switch over to another address book named anotherbook 
`ab list` | list out only addressbook and anotherbook
`ab switch addressbook` | switch over to existing address book name addressbook
`ab switch addressbook` | still remain on address book name addressbook
`ab switch anotherbook` | switch over to existing address book name anotherbook
`exit` | -
restart addressbook | boot up address book named anotherbook

### Checklist
